extensions/amp-story/1.0/bookend/components/article.js

```diff
@@ -99,19 +118,15 @@ export class ArticleComponent {
           </amp-img>`;
 
       addAttributesToElement(ampImg, dict({'src': articleData.image}));
-      template.appendChild(ampImg);
+      el.appendChild(ampImg);
     }
 
-    const heading =
-      html`<h2 class="i-amphtml-story-bookend-article-heading"></h2>`;
-    heading.textContent = articleData.title;
-    template.appendChild(heading);
+    const articleElements = htmlRefs(el);
+    const {heading, meta} = /** @type {!articleElsDef} */ (articleElements);
```

Comment:
Hmm, actually do you really need this typedef?

Reply:
I guess not! In a previous version, I got a "'heading or 'meta' is not defined" by the compiler when trying to use it on later lines. But not anymore 🤔